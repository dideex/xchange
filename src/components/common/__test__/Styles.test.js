import {
  coinColors,
  SvgCurrency,
  container,
  robotoSlab,
  H2,
  StatusIconColors,
  StatusTitles,
  MainSectionWrap,
} from '../Styles'

describe('Style snapshots', () => {
  it('coinColors', () => {
    expect(coinColors).toMatchSnapshot()
  })
  it('SvgCurrency', () => {
    expect(SvgCurrency).toMatchSnapshot()
  })
  it('container', () => {
    expect(container).toMatchSnapshot()
  })
  it('robotoSlab', () => {
    expect(robotoSlab).toMatchSnapshot()
  })
  it('H2', () => {
    expect(H2).toMatchSnapshot()
  })
  it('StatusIconColors', () => {
    expect(StatusIconColors).toMatchSnapshot()
  })
  it('StatusTitles', () => {
    expect(StatusTitles).toMatchSnapshot()
  })
  it('MainSectionWrap', () => {
    expect(MainSectionWrap).toMatchSnapshot()
  })
})
