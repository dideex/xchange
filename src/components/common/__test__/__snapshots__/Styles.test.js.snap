// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Style snapshots H2 1`] = `[Function]`;

exports[`Style snapshots MainSectionWrap 1`] = `[Function]`;

exports[`Style snapshots StatusIconColors 1`] = `
Array [
  "transparent",
  "#FFE712",
  "#ff9800",
  "#8FBE00",
  "#FC0000",
]
`;

exports[`Style snapshots StatusTitles 1`] = `
Array [
  "Не создан",
  "Ожидает перевода",
  "Ожидает подтверждения",
  "Переведено",
  "Отменено",
]
`;

exports[`Style snapshots SvgCurrency 1`] = `
Object {
  "height": "32px",
  "width": "32px",
}
`;

exports[`Style snapshots coinColors 1`] = `
Object {
  "body": Object {
    "fill": "#e2b255",
  },
  "label": Object {
    "fill": "#edda48",
  },
}
`;

exports[`Style snapshots container 1`] = `
"
  max-width: 1180px;
  width: 100%;
  margin: 0 auto;
  @media (max-width: 768px) {
    max-width: 360px;
  } 
  @media (max-width: 1024px) {
    max-width: 768px;
  } 
"
`;

exports[`Style snapshots robotoSlab 1`] = `
"'Roboto Slab', 'PT Serif', 'Monospaced Number', 'Chinese Quote', -apple-system,
BlinkMacSystemFont, 'Segoe UI', Roboto, 'PingFang SC', 'Hiragino Sans GB',
'Microsoft YaHei', 'Helvetica Neue', Helvetica, Arial, sans-serif"
`;
